packages/runtime/src/parser/parser.ts

```diff
@@ -882,6 +882,50 @@ export function parser(
 	return opCodeList;
 }
 
+// check algorand is auto added intcblock for optimize size contract
+export function isAddIntcblock(ops: Operator[], interpreter: Interpreter): boolean {
```

Comment:
Try to use arrow method here.

Reply:
Why we need to you arrow method?

Reply:
Okay ~~. We discussed, should use `function` keyword in this case